pkg/kopia/command/common.go

```diff
@@ -15,16 +15,34 @@
 package command
 
 import (
+	"os"
+
 	"github.com/kanisterio/kanister/pkg/field"
 	"github.com/kanisterio/kanister/pkg/log"
 	"github.com/kanisterio/kanister/pkg/logsafe"
 )
 
+const (
+	// LogLevelVarName is the environment variable that controls Kopia log level.
+	LogLevelVarName = "KOPIA_LOG_LEVEL"
+	// FileLogLevelVarName is the environment variable that controls Kopia file log level.
+	FileLogLevelVarName = "KOPIA_FILE_LOG_LEVEL"
+)
+
+func LogLevel() string {
+	return os.Getenv(LogLevelVarName)
```

Comment:
how is this  environment variable set? should we default to something if this env var is not set?

Reply:
In updated code, default value is provided.